### PR TITLE
Update editing_a_containers_provider.md

### PR DESCRIPTION
**Pull request by rwellon**
Pull request to add a note regarding the limitation for configuring Prometheus alerts for a container provider.

### CODE REVIEW

**Comment by rwellon** [comment]
@miq-bot assign @Fryguy

**Comment by rwellon** [comment]
@miq-bot assign @agrare

**Comment by rwellon** [comment]
@miq-bot add_labels petrosian/yes? oparin/yes? quinteros/yes?

**Comment by Fryguy** [comment]
@rwellon Do you need this backported to any specific versions?

File: managing_providers/_topics/editing_a_containers_provider.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Edit information about a provider such as the name, hostname, IP address or port
 
 7.  Under **Endpoints** in the **Alerts** tab, configure the following for Prometheus alerting from the cluster.
 
+    **Note:** Prometheus alerts are not supported on Red Hat OpenShift Container Platform version 4.
+
```
Comment (agrare) [inline managing_providers/_topics/editing_a_containers_provider.md:74]: @rwellon we no longer support OCPv3 at all, so I think we'd be better off deleting this entire section from the documentation

Comment (Fryguy) [inline managing_providers/_topics/editing_a_containers_provider.md:74]: The only version of OCP we support is 4, so this comment ends up creating a null situation.  I think it's better to just drop this entire section.

Note that we've dropped the entire OpenShift Alerts pages as well as the Monitor top-level menu for this reason.  Can you see if there are other docs to drop as well related to that?

**Comment by agrare** [comment]
We still had the alerts endpoint in the container_manager create form, https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/506

**Comment by agrare** [comment]
@rwellon it looks like you tried to merge master into this branch, as a project we rebase PR branches on master rather than merging master into the branch just to keep the history clean and not have more merge commits than necessary.

You can refer to the git workflow https://www.manageiq.org/docs/guides/developer_setup/git_workflow specifically `git config --global --bool pull.rebase true` or you can `git fetch upstream; git rebase upstream/master`

**Comment by rwellon** [comment]
Closing this PR - will remove content in new PR

**Pull request closed without merging**